src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -146,7 +161,7 @@ private HostAndPort initSentinels(Set<String> sentinels, final String masterName
 
       Jedis jedis = null;
       try {
-        jedis = new Jedis(hap);
+        jedis = new Jedis(hap.getHost(), hap.getPort(), sentinelConnectionTimeout, sentinelSoTimeout, sentinelPassword);
```

Comment:
Without sending password and clientName in constructor, auth and clientSetName can be done here.

Reply:
True. I'll make that change.

Reply:
I've followed the same approach as `JedisFactory` in configuring Jedis for connecting to sentinel (i.e. when respective value is not null).